app.js

```diff
@@ -0,0 +1,137 @@
+// Usage:
+// node app.js -r -c config.yaml -u "http://localhost:9001" # remember to add the registration!
+// node app.js -p 9001 -c config.yaml
+var qs = require('querystring');
+var requestLib = require("request");
+var yaml = require("yamljs");
+var bridge;
+
+function startServer(config, rooms, callback) {
+    if ("tls" in config) {
+        var fs = require("fs");
+        var tls_options = {
+            key: fs.readFileSync(config["tls"]["key_file"]),
```

Comment:
You may find `config.tls.key_file` nicer to write, and you know it's defined because of your schema.